extensions/amp-next-page/amp-next-page.md

```diff
@@ -163,82 +148,173 @@ Elements which are common across multiple pages can be programmatically
+  <h2>I replaced the first instance of my type (sticky-123)</h2>
+</div>
+```
 
-| Key                | Value                                                             |
-| ------------------ | ----------------------------------------------------------------- |
-| `pages` (required) | Ordered array of one or more page objects                         |
-| `hideSelectors`    | Optional array of string CSS selectors to hide in child documents |
+### Inline Configuration
+
+The inline configuration defines the documents recommended by `amp-next-page` to
+the user as an ordered array of page objects in JSON format.
 
 Each page object should have the following format:
 
-| Key                   | Value                                                                                                                                                            |
-| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `ampUrl` (required)   | String URL of the page. Must be on the same origin as the current document. URLs will automatically be rewritten to point to the Google AMP cache when required. |
-| `title` (required)    | String title of the page, will be used when rendering the recommendation box                                                                                     |
-| `imageUrl` (required) | String URL of the image to display in the recommendation box                                                                                                     |
+| Key                | Value                                                                                                                                                            |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `url` (required)   | String URL of the page. Must be on the same origin as the current document. URLs will automatically be rewritten to point to the Google AMP cache when required. |
+| `title` (required) | String title of the page, will be used when rendering the recommendation box                                                                                     |
+| `image` (required) | String URL of the image to display in the recommendation box                                                                                                     |
+
+To enable for an infinite loading experience without the need for a server-side configuration (remote-loading), `amp-next-page` automatically enables `deep-parsing` (see Attributes), which allows it to parse more suggestions by recursively looking at inline configurations inside `<amp-next-page>` tags on the loaded documents. To disable this behavior, set `deep-parsing` to `false`.
+
```

Comment:
I'm trying to figure out where the information in this section best belongs, but I'm a little confused.
- We say "`amp-next-page` automatically enables `deep-parsing`" but `deep-parsing` is an optional attribute? What does this mean? 
- What is the default behavior (no optional attributes)? Do we show the "recommendation box" or is one article loaded after another?
- What happens if I only have one article in my configuration when the  "recommendation box" is shown? Will it only show that one recommendation or will it just auto load that page?

Reply:
@wassgha I still don't think these questions are answered.